docs/contract-test-collection.md

```diff
@@ -309,109 +747,253 @@ storage (pair
+
 ```js
-storage nat;
-```  
+storage nat
+```
+
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+storage: 1
+```
+
+</TabItem>
+</Tabs>
+
 ## ContractTopLevelViews
 
 This contract has a series of views which are sections of Michelson Code, for example:
 
 ```js
 view "add" nat nat { UNPAIR ; ADD } ;
 ```
+
 which can be called by other contracts to calculate and return some value.
 
 [KT1KJbtwARB2w3yN5fRjBE6CXJh5t9Hbbbpv](https://better-call.dev/hangzhou2net/KT1KJbtwARB2w3yN5fRjBE6CXJh5t9Hbbbpv/code)
 
 #### Entrypoints
-  * default
 
-#### Storage:
+- default
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'}
+]}>
+<TabItem value="michelson">
+
+```js
+storage nat
+```
+
+</TabItem>
+  <TabItem value="taquito">
+
 ```js
-storage nat ;
-```  
+storage: 1
+```
+
+</TabItem>
+</Tabs>
+
 # Tzip-7 Contracts
+
 Tzip-7 introduced the approvable ledger: [Tzip-7](https://gitlab.com/tezos/tzip/-/blob/master/proposals/tzip-7/tzip-7.md)
 
 ## TokenContract
 
 [KT1KxMdSBa9ippgTWwU7d1FkzTF5CirUkcp4](https://better-call.dev/hangzhou2net/KT1KxMdSBa9ippgTWwU7d1FkzTF5CirUkcp4/code)
 
-- [A Beginner's Guide to Tezos Tzip-7 Prooposal](https://claudebarde.medium.com/a-beginners-guide-to-tezos-tzip-7-proposal-90a8b816af7e)
+- [A Beginner's Guide to Tezos Tzip-7 Proposal](https://claudebarde.medium.com/a-beginners-guide-to-tezos-tzip-7-proposal-90a8b816af7e)
+
+This contract has an FA1.2 interface. To determine if a contract has an FA1.2 interface we can use
 
-This contract has an FA1.2 interface.  To determine if a contract has an FA1.2 interface we can use 
 ```
 tezos-client check contract KT1CfFBaLoUrgv93k8668KCCcu2hNDNYPz4L implements fa1.2
 ```
 
 #### Entrypoints:
-  * approve
-  * burn
-  * getAdministrator
-  * getAllowance
-  * getBalance
-  * getTotalSupply
-  * mint
-  * setAdministrator
-  * setPause
-  * transfer
-
-#### Storage:
+
+- approve
+- burn
+- getAdministrator
+- getAllowance
+- getBalance
+- getTotalSupply
+- mint
+- setAdministrator
+- setPause
+- transfer
+
+<Tabs
+defaultValue="michelson"
+values={[
+{label: 'Michelson Storage', value: 'michelson'},
+{label: 'Taquito Storage', value: 'taquito'}
+]}>
+<TabItem value="michelson">
+
 ```js
 storage (pair (big_map address (pair nat (map address nat)))
               (pair address (pair bool nat)));
 ```
 
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+    export const tokenStorage = (k: string) => ({
+      "prim": "Pair", "args": [[
+        {
+          prim: 'Elt',
+          args: [
+            { string: k },
+            { prim: 'Pair', args: [{ "int": "2" }, []] }
+          ]
+        }
+      ], { "prim": "Pair", "args": [{ "string": k }, { "prim": "Pair", "args": [{ "prim": "False" }, { "int": "200" }] }] }]
+    });
+
+    const op = await tezos.contract.originate({
+      balance: '1',
+      code: tokenCode,
+      storage: tokenStorage(await tezos.signer.publicKeyHash()),
```

Comment:
We can not write the storage in michelson when deploying a contract with Taquito using the storage property/
The storage should be:
```
const pkh = await Tezos.signer.publicKeyHash();
const bm = new MichelsonMap();
bm.set(pkh, {
    0: "2",
    1: new MichelsonMap()
});

storage: {
    0: bm,
    1: pkh,
    2: true,
    3: "200"
}
```

Reply:
I made the change, but i put bigMap instead of bm